docs/contract.yaml

```diff
@@ -0,0 +1,156 @@
+openapi: 3.0.0
+info: 
+  title: Spec Math API Server
+  version: 1.0.0
+  description: An API that exposes some operations from the Spec Math Library
+servers:
+  - description: Spec Math API Server
+    url: http://specmath.cloud.google.com/v1
+paths:
+  /specMath/merge:
+    post:
+      description: |
+      perform a union on two specs, with optional defaults file and conflict resolutions.
+      The endpoint may return a success, which is the result of a successful union,
+      or a failure, which means some unhandled conflicts occurred in the union. In the latter case,
+      a 406 code will be sent along with an array of conflict objects. The user now has two ways
+      to resolve the conflicts. First, they may iterate through the conflicts and add a value to the
+      resolvedValue property based on the options available. They could also manually resolve the
+      conflicts in the original YAML files. In either case, they can send another request to this
+      endpoint with the conflicts resolved.
+      operationId: specMathUnion
+      requestBody:
+        description: data needed for the union operation
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/MergeBody"
+      responses: 
+        '200':
+          description: operation was successful, returns the resultant spec 
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/OperationSuccess"
+        '406':
```

Comment:
I think 406 is not appropriate in this context.  Probably best to merge this in with the 200 response.  The call to the server succeeded and returned, but the specs just couldn't be merged. That feels more like a status in the response payload.

Reply:
I agree - Just looking at [this](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406), "In practice, this error is very rarely used." 406 seems to be more related to HTTP-level problems than semantics of APIs.